src/lib/KeepaliveManager.ts

```diff
@@ -41,13 +42,15 @@ export default class KeepaliveManager {
 			'clear' in variant
 				? variant
 				: getTimer(variant)
-		this.setKeepalive(client.options.keepalive)
+		if (client.options.keepalive !== undefined) {
+			this.setKeepalive(client.options.keepalive)
+		}
 	}
 
 	private clear() {
-		if (this.timerId) {
+		if (this.timerId !== undefined) {
```

Comment:
Not strictly a TypeScript issue, but ESlint will complain if we use a strict configuration, as nullish comparisons can lead to problematic bugs.

Reply:
what if timerId is null?